maven-api-impl/src/main/java/org/apache/maven/internal/impl/model/MavenBuildTimestamp.java

```diff
@@ -26,18 +26,18 @@
 import java.util.Properties;
 import java.util.TimeZone;
 
+import org.apache.maven.api.Constants;
+
 /**
  * MavenBuildTimestamp
  */
 public class MavenBuildTimestamp {
     // ISO 8601-compliant timestamp for machine readability
-    public static final String DEFAULT_BUILD_TIMESTAMP_FORMAT = "yyyy-MM-dd'T'HH:mm:ss'Z'";
-
-    public static final String BUILD_TIMESTAMP_FORMAT_PROPERTY = "maven.build.timestamp.format";
+    public static final String DEFAULT_BUILD_TIMESTAMP_FORMAT = "yyyy-MM-dd'T'HH:mm:ssXXX";
```

Comment:
Why not use `Constants.MAVEN_BUILD_TIMESTAMP_FORMAT` instead of a literal again?

Reply:
`Constants.MAVEN_BUILD_TIMESTAMP_FORMAT` defines the property name.  The default value is available on the `@Config` annotation, but at this point, it is for documentation only.  

Another way would be to move the default values as constants, i.e. move DEFAULT_BUILD_TIMESTAMP_FORMAT to `Constants` and link both there.

Reply:
Yes, that seems to make sense. No need to define it twice, yes?

Reply:
I've had a look, unfortunately, most of those properties don't really use a simple value.
A bunch of them use property substitution such ac `${maven.home}/conf`, or even non computable values, such as `cores/2 + 1`.  It would only work for a few of them, so I'd rather keep it that way.

Reply:
That's fine.